elements/rh-number/rh-number.story.js

```diff
@@ -1,94 +1,94 @@
-import { storiesOf } from "@storybook/polymer";
```

Comment:
I'm guess this was done back before the third-party dependencies issue was figured out.  That is resolved now, so if you merge master into your PR branch you shouldn't have to comment out rh-number's story in order to get storybook working.